ignite/src/main/java/org/hibernate/ogm/datastore/ignite/impl/IgniteCacheInitializer.java

```diff
@@ -149,6 +154,8 @@ private void initializeIdSources(SchemaDefinitionContext context, IgniteDatastor
 	private CacheConfiguration createCacheConfiguration(IdSourceKeyMetadata idSourceKeyMetadata) {
 		CacheConfiguration result = new CacheConfiguration();
 		result.setName( StringHelper.stringBeforePoint( idSourceKeyMetadata.getName() ) );
+		result.setAtomicityMode( CacheAtomicityMode.TRANSACTIONAL );
+		result.setCacheMode( CacheMode.REPLICATED );
```

Comment:
it is unacceptable! 
Why you configure default CacheMode

Reply:
Could you please remove it to configuration if you need configure CacheMode.
And ... What will with other CacheMode?

Reply:
Why "it is unacceptable"? I think that replicated mode is more suitable for ID cache than partitioned